components/FormInput/FormInput.css

```diff
@@ -48,15 +51,22 @@
 
 .containerActive {}
 
+.containerFocus {
+  box-shadow: inset 0 0 0 1px var(--color-blue), 0 0 2px color(var(--color-blue) l(70%));
+}
+
 .containerError {
   box-shadow: inset 0 0 0 1px var(--color-error);
   border-radius: 4px 4px 0 0;
 }
 .containerActive .label {
-  transform: scale(0.8) translate(-0.75em, -2.8em);
```

Comment:
Yeah it could certainly do with some improvements. It's that age-old issue of transitioning height.

Not gonna do anything on it for this PR though.